decoder/src/test/java/openlr/decoder/data/OpenLRRatingImplTest.java

```diff
@@ -236,6 +255,84 @@ public final void testRating3() {
         }
     }
 
+    /**
+     * Rating test of junction node when applying no non-junction node factor
+     */
+    @Test
+    public final void testRatingJunctionNodeNoNonJunctionNodeFactor() {
+
+        try {
```

Comment:
Would it harm to let the test framework catch an unexpected exception? Leaving out `try {} catch () {}` may improve readability.

Reply:
Good point.  I just copied the pattern of the other tests in the class without thinking.  But you are right, it's better to just throw the exception.  I will change it.